### ISSUE TITLE
drone delivery code stops in polled mode guided

### ISSUE THREAD

**Issue by prashanthravi19** [dronekit/dronekit-python#1071]
While running the drone delivery app the polled mode stops in guided and it continues to repeat command without moving froward

**Comment by techwithomar** [comment]
same problem here

**Comment by tarunsai-07** [comment]
yeah it's just repeating polled mode: GUIDED but the drone is not moving at all

**Comment by PGSK1234** [comment]
I am stuck at same issue... is there a resolution for this please?
